tests/aggregation_regress/tests.py

```diff
@@ -1450,8 +1452,10 @@ def test_ticket_11293(self):
         query = Book.objects.annotate(Count('authors')).filter(
             q1 | q2).order_by('pk')
         self.assertQuerysetEqual(
-            query, [1, 4, 5, 6],
-            lambda b: b.pk)
+            query,
+            [self.b1.pk, self.b4.pk, self.b5.pk, self.b6.pk],
```

Comment:
Book PKs are assumed to be in order, which may not be true on CockroachDB. 
```suggestion
            sorted([self.b1.pk, self.b4.pk, self.b5.pk, self.b6.pk]),
```

Reply:
added in amended commit. Thanks for catching this!